.github/workflows/build.yml

```diff
@@ -23,16 +26,16 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          persist-credentials: false
       - name: Configure tree
         run: |
           MAKE_ARG=-j XARCH=x64 CONFIG_ARG='--enable-flambda --enable-cmm-invariants --enable-dependency-generation --enable-native-toplevel' OCAMLRUNPARAM=b,v=0 bash -xe tools/ci/actions/runner.sh configure
       - name: Build
         run: |
           MAKE_ARG=-j bash -xe tools/ci/actions/runner.sh build
       - name: Prepare Artifact
-        run: |
-          git config --local --unset http.https://github.com/.extraheader
```

Comment:
This line is no longer necessary with the `persist-credentials` above (and, it turns out, `git` exits with code 5 if you unset a configuration value which isn't set)